src/zkevm_specs/state.py

```diff
@@ -388,23 +391,33 @@ def check_tx_log(row: Row, row_prev: Row):
     if row.tag() == row_prev.tag():
         if tx_id != pre_tx_id:
             assert tx_id == pre_tx_id + 1
-            assert log_id == 0
+            # in RW table, log_id starts with 1
+            assert log_id == 1
+            # first field_tag is Address when tx changes
```

Comment:
Could you add some comment why this need to be so?

Reply:
added in [75fb5f7](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/220/commits/75fb5f730d5659ddfb7c40b8c3cc8256857efac9) thanks !!